test/src/406-ducc-webhook-notifications/main

```diff
@@ -0,0 +1,96 @@
+cvmfs_test_name="Webhook notifications test"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="ducc"
+
+cleanup() {
+  pid=$(ps aux | grep 'registry-webhook.py' | awk -F' ' '{print $2}' )
+  kill $pid
+}
+
+check_port() {
+    if nc -zv 127.0.0.1 8080 2>&1;
```

Comment:
Let's make this as a loop without recursion, and let's wait for a second between the retries

Reply:
Can I use sleep here then?

Reply:
Yes, I think that's all right as a means to throttle the rate of `nc` calls